woocommerce/woocommerce-android
# Track payment cancelation and rename payment flow start event

Closes: #6786


### Description

* Renames `CARD_PRESENT_COLLECT_PAYMENT_TAPPED` -> `PAYMENTS_FLOW_ORDER_COLLECT_PAYMENT_TAPPED`
* Renames `SIMPLE_PAYMENTS_FLOW_CANCELED` -> `PAYMENTS_FLOW_CANCELED`
* Adds `PAYMENTS_FLOW_CANCELED` tracking when
    * Selection payment method fragment is closed during order payment flow (arg `flow=order_payment`)
    * Back click on the main simple payment fragment (arg `flow=simple_payment`)

### Testing instructions

* Use logcat with filtering by `Tracked`
* Go via Simple payments flow and Order payment flow and notice tracked events
